api/v1alpha1/openapi.yaml

```diff
@@ -5044,4 +5147,168 @@ components:
           type: string
       required:
         - version
-
+    Event:
+      type: object
+      required:
+        - kind
+        - apiVersion
+        - id
+        - timestamp
+        - type
+        - source
+        - status
+        - message
+        - severity
+        - correlationId
```

Comment:
Why is correlationId required? Can't there be events that are not correlated to other events ?

Reply:
In that case there will be a single event with that ID.  When the system emits an event it doesn't know if other correlated events will come after it.